.github/workflows/e2e_cron.yml

```diff
@@ -47,7 +47,7 @@ jobs:
       - name: Run Playwright tests - ${{ inputs.environment != null && inputs.environment || 'stage' }}
         if: github.actor != 'dependabot[bot]'
         run: npm run e2e
-        timeout-minutes: 10
+        timeout-minutes: 20
```

Comment:
Not sure if this is https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idtimeout-minutes with its default value of 360 minutes (SIX hours?!)

But :+1:, should work until the suite gets big enough to change this to 30..40...

Reply:
Exactly.. this isn't a "solution". I think Raphael put this TO in intentionally to get alerts so we can push for optimization...

Reply:
Have we measure to see where the bottleneck(s) are now? Assuming we're already running anything concurrently that we can, I'd like to see what is slowing it down and speed it up if possible. This is one area where mocking external systems (fxa, hibp, onerep, etc) could be helpful if they are the problem.